File: PHP/Baju.php
<?php
require_once "Aksesoris.php";

class Baju extends Aksesoris {
    protected $untuk_hewan;
    protected $size_baju;
    protected $merk_baju;

    public function __construct($id = 0, $nama = "", $harga = 0.0, $stok = 0, $foto = "default.jpg", $jenis = "", $bahan = "", $warna = "", $untuk = "", $size = "", $merk = "") {
        parent::__construct($id, $nama, $harga, $stok, $foto, $jenis, $bahan, $warna);
        $this->untuk_hewan = $untuk;
        $this->size_baju = $size;
        $this->merk_baju = $merk;
    }

    public function getUntuk() { return $this->untuk_hewan; }
    public function getSize() { return $this->size_baju; }
    public function getMerk() { return $this->merk_baju; }
}
?>

File: PHYTON/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id=0, nama="", harga=0.0, stok=0, jenis="", bahan="", warna="", untuk="", size="", merk=""):
        super().__init__(id, nama, harga, stok, jenis, bahan, warna)
        self._untuk_hewan = untuk
        self._size_baju = size
        self._merk_baju = merk
    
    def set_untuk(self, untuk):
        self._untuk_hewan = untuk
    
    def set_size(self, size):
        self._size_baju = size
    
    def set_merk(self, merk):
        self._merk_baju = merk
    
    def get_untuk(self):
        return self._untuk_hewan
    
    def get_size(self):
        return self._size_baju
    
    def get_merk(self):
        return self._merk_baju